lab_3/main.py

```diff
@@ -168,7 +156,7 @@ class NGramTrie:
     """
 
     def __init__(self, n: int, letter_storage: LetterStorage):
-        self.size = n
+        self.size = 2
```

Comment:
It should not be constant - it should be passed as an argument when an instance is created.

Reply:
Done